person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    @age >= 18 || @parent_permission ? true : false
```

Comment:
- [ ] The private method of_age is supposed to be used, not the variable or re-writing the method again so it would be something like: `of_age?() || @parent_permission`, the true or false is not totally needed, though it would work. It would also work without it because if both conditions are false, the it returns false automatically.